examples/scenes/src/images.rs

```diff
@@ -43,7 +43,7 @@ impl ImageCache {
 }
 
 fn decode_image(data: &[u8]) -> anyhow::Result<Image> {
-    let image = image::io::Reader::new(std::io::Cursor::new(data))
+    let image = image::ImageReader::new(std::io::Cursor::new(data))
```

Comment:
I was going to do this one as a separate PR ... if you don't mind. (Been looking at what'll happen as we get ready to release a new Vello...)

Reply:
Sure! This was just to unblock me as I needed to use `cargo update`

Reply:
Done in #682.